src/index.js

```diff
@@ -45,11 +46,10 @@ async function main(req, context) {
     AWS_S3_REGION, AWS_S3_ACCESS_KEY_ID, AWS_S3_SECRET_ACCESS_KEY,
     HTTP_TIMEOUT_EXTERNAL,
   } = env;
-  const { searchParams } = new URL(req.url);
-  const params = Object.fromEntries(searchParams.entries());
+
   const {
     owner, repo, ref, path, prefix = 'live',
-  } = params;
+  } = context.data;
```

Comment:
As it turns out, `context.data` contains the querystring params *even* when the request has no body